corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -269,7 +269,8 @@ def _get_vellum_plugins(domain, form, module):
     privileges.
     """
     vellum_plugins = ["modeliteration", "itemset", "atwho"]
-    if toggles.COMMTRACK.enabled(domain):
+    if (toggles.COMMTRACK.enabled(domain)
+            or toggles.NON_COMMTRACK_LEDGERS.enabled(domain)):
         vellum_plugins.append("commtrack")
```

Comment:
@dimagi/team-form-designer 
This seemed to do the trick for enabling transaction question types in Vellum - do you know if there are any possibly unintended side-effects?  Are there other commtrack features besides transaction questions?  Should we split those up and make a `"ledgers"` vellum plugin?

Reply:
@esoergel interesting. I've never seen that team name. looks like it's just cory and clayton on there.Tagging @millerdev since he would know the most about this.

I think this is fine and we don't need a ledgers vellum plugin as it only adds the questions.

It does seem unfortunate that we have two feature flags controlling this. Is there a release path for the non commtrack ledgers? could it be made into an app manager add on for advanced plans ( @orangejenny ?) ? also is there a reason the previous check wasn't `domain.commtrack_enabled`?

Reply:
The `commtrack` plugin adds four question types: Balance, Transfer, Dispense, and Receive. All are closely related, so I'm assuming that's what you wanted. LGTM

Reply:
> could it be made into an app manager add on for advanced plans

That'd make it visible to anyone (with an advanced plan). It's fine technically, if we aren't concerned about random SaaS users turning it on.

Reply:
I think this would be a valuable tool for complicated enterprise apps, and would very much like to see it in product.  That said, I'm guessing we'd want product to weigh in and maybe revisit what the fields are named and how we expect this to be used before releasing this - the language is focused around stock updates, so it is pretty nonsensical as a generic tool.

This flag is used in two places - to decide whether to display the question types, and to decide whether to sync ledgers.  In the former case, an addon seems fine once this is released, and in the latter case, I guess we could instead do a check for stock states on the domain or something.  I suppose my ideal state for this would be to consider ledgers a feature independent from CommCare Supply, but enabled when you turn on Supply.  Not sure that's worth fully decoupling at this point, but agree that it'd be useful if/when we push this beyond eNikshay.

Reply:
We definitely don't want any new non-dimagi's using this for now.